src/code/code_800EC960.c

```diff
@@ -1267,7 +1267,8 @@ OcarinaNote sScarecrowsLongSongSecondNote;
 u8 sAudioHasMalonBgm;
 f32 sAudioMalonBgmDist;
 
-void PadMgr_RequestPadData(PadMgr* padmgr, Input* inputs, s32 mode);
+void PadMgr_RequestPadData(PadMgr* padMgr, Input inputs[MAXCONTROLLERS], s32 gameRequest);
```

Comment:
We generally don't use the array notation in arguments, and I personally think we should continue to avoid it. This was discussed recently in discord and I explained why I think it's bad, so it might be simpler to just read that for my opinion (see <https://discord.com/channels/688807550715560050/688851976746041391/997577684446093364> or search for "array argument" in discord)